libs/domain/content/CHANGELOG.md

```diff
@@ -3,6 +3,23 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+# 1.1.0 (2023-09-22)
+
+
+### Bug Fixes
+
+* blank screen instead of 404 page ([#788](https://github.com/spryker/oryx/issues/788)) ([720eb12](https://github.com/spryker/oryx/commit/720eb12bea26167de0868218f445429ebb3e8e8e))
+
+
+### Features
```

Comment:
features first would be great, can you also update the feature commit titles here and all other sub packages?